packages/core/src/components/component.ts

```diff
@@ -90,10 +90,28 @@ export class Component {
 				"style",
 				"prefix"
 			);
-			return DOMUtils.appendOrSelect(
-				this.parent,
-				`g.${settings.prefix}--${chartprefix}--${this.type}`
-			);
+
+			// @todo Chart type equals to axis-chart
+			if (
+				this.type === "line" ||
+				this.type === "scatter" ||
+				this.type === "area" ||
+				this.type === "bubble" ||
+				this.type === "area-stacked" ||
+				this.type === "grouped-bar" ||
+				this.type === "simple-bar" ||
+				this.type === "stacked-bar" ||
+				this.type === "scatter-stacked"
```

Comment:
yes this doesn't make sense

Reply:
Thanks for your comment.
I use a flag in getContainerSVG() to avoid this problem.